nomad/fsm.go

```diff
@@ -1156,33 +1158,35 @@ func (n *nomadFSM) applyCSIVolumeDeregister(buf []byte, index uint64) interface{
 	return nil
 }
 
-func (n *nomadFSM) applyCSIVolumeClaim(buf []byte, index uint64) interface{} {
```

Comment:
note for review: it's worth looking at this file without the diff... the diff really tangled this up because the `applyCSIVolumeBatchClaim` is similar to the `applyCSIVolumeClaim`. This should be entirely an addition.